src/picongpu/include/simulation_defines/param/fileOutput.param

```diff
@@ -63,6 +66,12 @@ namespace picongpu
             CreateEnergyDensityOperation<bmpl::_1>
             >::type EnergyDensity_Seq;
 
+    /* EnergyPerpendicular section: define longitudinal as 1=Y */
```

Comment:
please add a more detailed description on how to set the longitudinal direction e.g.: `x=0, y=1, z=2` given as boost-mpl int as second argument to `CreateEnergyPerpendicularOperation`.